samples/metrics_monitor/sample/cttmetrics_sample.cpp

```diff
@@ -45,28 +54,68 @@ void signal_handler(int signo)
     }
 }
 
+static void
+usage(const char *appname)
+{
+    printf("metrics_monitor - Monitors GPU usage per engine\n"
+        "\n"
+        "Usage: %s [OPTION]\n"
+        "\n"
+        "\tThe following parameters are optional:\n\n"
+        "\t[-h]          Show this help text.\n"
+        "\t[-s <num>]    Number of metric samples to collect during sampling period(valid range %u..%u, default %u).\n"
+        "\t[-p <ms>]     Sampling period in milliseconds(valid range %u..%u, default %u).\n"
+        "\t[-d <path>]   Path to gfx device (like /dev/dri/card* or /dev/dri/renderD*).\n"
+        "\t              If device is not set, the tool uses render node device with smaller number."
```

Comment:
Will the tool search for the first Intel adapter or just try renderD128?

Reply:
The tool does what's written in help output,  it uses render node device with smallest number. "smaller" is a misprint and will be fixed.

Reply:
Actually no. You have this line to initialize device:
```
g_ctx.gem_fd = device ? open(device, O_RDWR) : drmOpenWithType("i915", NULL, DRM_NODE_RENDER);
```

This means that by default you search for the first Intel GPU render node. You need to clarify what's the used algo and what means "first".

Reply:
I DID NOT use word 'first' intentionally. Where did you find it?

Reply:
you do not mention that it will look for the Intel GPU either. I read this that it can found other vendor render node and fail to work.

Reply:
added i915